packages/manager/modules/netapp/src/dashboard/volumes/dashboard/acl/routing.js

```diff
@@ -0,0 +1,44 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('netapp.dashboard.volumes.dashboard.acl', {
+    url: '/acl',
+    component: 'ovhManagerNetAppVolumesDashboardAcl',
+    resolve: {
+      acls: /* @ngInject */ ($http, serviceName, volumeId) =>
+        $http
+          .get(`/storage/netapp/${serviceName}/share/${volumeId}/acl`)
+          .then(({ data }) => data)
+          .catch(() => []),
```

Comment:
Should we stil display view if acls returns an error, in that case I'm not sure about this behaviour

Reply:
You want me to redirect instead to the volume dashboard with an error message?

Reply:
Or let the error page handle it